tf_agents/pyproject.toml

```diff
@@ -9,7 +9,6 @@ license = "Apache-2.0"
 python = ">=3.7,<3.10"
 cogment-verse = {path = "../base_python", develop = true}
 tensorflow = {version="2.6.2", optional = true}
-torch = {version="1.8.1", optional = true}
```

Comment:
Don't forget to run `poetry update` in the `tf_agents` folder to update the lock file and commit it.